packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -411,10 +446,17 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
                 dismissable={false}
                 icon={<StyledSvgWarning width={24} height={24} />}
                 description={
-                  <>
-                    <Text variant="t200">After issuing this as an NFT </Text>
-                    <YellowText variant="t200">editing options of this video will be disabled</YellowText>
-                  </>
+                  !videoFieldsLocked ? (
+                    <>
+                      <Text variant="t200">After issuing this as an NFT </Text>
+                      <YellowText variant="t200">editing options of this video will be disabled</YellowText>
```

Comment:
I think this is my code, but I'm pretty sure that we can put YellowText inside Text i.e. `<Text>< YellowText></Text>`. Could you do that?